Cargo.toml

```diff
@@ -64,7 +64,7 @@ log = { version = "0.4", features = [
 ] }
 mockall = "0.12"
 mozsvc-common = "0.2"
-openssl = { version = "0.10" }
+openssl = { version = "0.10.70" }
```

Comment:
We tend to not specify minor revisions here. It's a good balance of not having to touch `Cargo.toml` for every minor bump + allows cargo to figure out the latest version for us when needed -- which shouldn't matter to our code as minor revisions should be backwards compat/not change APIs.

To handle CVE/RUSTSECs in this way you can _usually_ just do an e.g. `cargo update -p openssl` to bump to the latest `0.10` only in `Cargo.lock`

Reply:
IIRC, [Rust is a bit odd about versions](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#version-requirement-syntax), so 
`0.10.70` translates to "use anything between 0.10.70 and 0.11.0" (which is the same as `~0.10.70` and because there's a leading `0` it's special cased.)

If we need to lock to a specific version, we should use `^0.10.70`

Reply:
Cool, thx for that! Will do.